src/context.ts

```diff
@@ -89,7 +89,7 @@ export class Context<
   private _pH: Record<string, string> | undefined = undefined // _preparedHeaders
   private _res: Response | undefined
   private _path: string = '/'
-  private _pData?: Record<string, string> | null // __paramData
+  private _params?: Record<string, string> | null // __params
```

Comment:
The comment `// __params` is not necessary.

Reply:
Fixed!
0e39109